src/server/migrations/1618340375981-RemoveViewDependencies.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+/**
+ * This migration replaces the ScheduleEntryView created in
+ * 1603825601596-ScheduleViews.ts to remove the relation between the
```

Comment:
I would add square brackets round this, so that it can be clicked through in the documentation. That's also true for the other views, but that's probably less less important than having a reference to the other migration.

```suggestion
 * [[1603825601596-ScheduleViews.ts]] to remove the relation between the
```

Reply:
Not a bad idea, but the migrations are actually [ignored by our typedoc config](https://github.com/seas-computing/course-planner/blob/main/typedoc.json#L6) so they won't show up in the generated docs anyway.